session_4/blogspot/app/views/articles/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<br>
+<% unless notice.to_s.empty? %>
+  <div class="alert alert-success" role="alert">
+    <%= notice %>
+  </div>
+<% end %>
+<br>
+<font color="#3385ff" face="WildWest">
```

Comment:
A better way of defining such styles would be to use css file.

`app/assets/stylesheets/application.scss`:
```css
h1.brand-text {
  font-face: 'WildWest',
  color: '#3385ff'
}
```

and then in your views:

```erb
<h1 class="text-center brand-text">Blogspot</h1>
```

Take your time learning the basics of css: https://www.w3schools.com/css/default.asp